.vscode/launch.json

```diff
@@ -56,6 +56,27 @@
       "type": "coreclr",
       "request": "attach",
       "processId": "${command:pickProcess}"
+    },
+    {
+      "name": "Debug Frontend Tests",
```

Comment:
This allows us to debug the currently open test file.  Regrettably it does run all the tests in the file instead of only the one you want.